extensions/azurecore/src/azureResource/utils.ts

```diff
@@ -347,24 +340,30 @@ export async function makeHttpRequest(account: azdata.Account, subscription: azu
 		validateStatus: () => true // Never throw
 	};
 
-	let response;
+	let requestUrl: string;
+	if (host) {
+		requestUrl = `${host}${path}`;
```

Comment:
I'd suggest looking into using the URI class for making the full URL - that's safer and should handle either starting with a / or not. 

https://github.com/microsoft/azuredatastudio/blob/main/src/vs/vscode.d.ts#L1306

(test it out - if it doesn't handle starting with or without / then probably not worth it though)